.github/ami-cleaner.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
```

Comment:
mmm just a thought, maybe this should all go with `set -e` to avoid that any aws call/escaping thing goes bad

Reply:
good point